src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const advanceDiscount = 50;
+  const regularDiscount = 20;
+  const res = rentPerDay * days;
+
+  if (days > 2) {
+    return days > 6 ? res - advanceDiscount : res - regularDiscount;
```

Comment:
The condition `days > 6` is also using a literal instead of a constant. It's recommended to define a constant for the minimum number of days required to receive an advance discount, as per the checklist.